xbmc/music/Album.h

```diff
@@ -71,6 +71,8 @@ class CAlbum
   }
 
   std::string GetArtistString() const;
+  const std::vector<std::string>& GetAlbumArtist() const;
+  const std::vector<std::string>& GetMusicBrainzAlbumArtistID() const;
```

Comment:
You can't return a reference to a stack variable either. Changing the return to just `std::vector<std::string>` fixes that.